string nodeName
uint8[] cuuid       # Byte array of length 16
uint8  componentType
---
uint8  status
